src/App.js

```diff
@@ -1,11 +1,11 @@
-import './App.css';
 import {
   BrowserRouter as Router, Routes,
```

Comment:
- [ ] Kindly remove the `as` and replace it with a coma `,` so you have `import { BrowserRouter, Router, Route } from 'react-router-dom';` kindly debug your app before re-submitting it for review